### ISSUE TITLE
Mobile DeFi: Prize for Best Celo Learn Tutorial!

### ISSUE THREAD

**Issue by ClaytonFigmentNetworks** [figment-networks/datahub-learn#30]
### **Brief**

The [Celo Learn Pathway](https://learn.figment.io/network-documentation/celo/celo-pathway) by Figment is the fastest way for new developers to learn how to build mobile-first applications on Celo. We give developers the documentation and tools they need to build on Celo while rewarding them in cUSD for completing tutorials. [Our tutorials](https://learn.figment.io/network-documentation/celo/tutorial) start from the beginning teaching developers how to connect to Celo via [DataHub](https://figment.io/datahub/celo/), our highly available full node infrastructure. As developers complete more tutorials, they learn how to query the Celo network, send transactions, and write their own Celo smart contracts. 

### **The** **Challenge**

We are looking to add more advanced Celo tutorials to Figment Learn. We have a community eager to learn more about building on Celo and we are looking to the Mobile DeFi hackathon community for help! 

- The person or team to create the best Celo tutorial for Figment Learn will win **$2,500 in cUSD**. 
- 2nd and 3rd place teams will win **$1,500 and $1,000 in cUSD** and all winning teams will win **DataHub Plus for 3 months**. 
- On top of that, we will be giving **$500 in cUSD** for any tutorial that passes the submission requirements. 

Check out our [Celo tutorials on Figment Learn](https://learn.figment.io/network-documentation/celo/tutorial) to get a sense of what we are looking for.

### **Judging Criteria**

A panel of Figment employees and Celo Figment Learn Community members will be judging which submitted tutorials are the best. 

**Your submission will be graded by:** 

- Quality of Information
- Skill Level (intermediate or Advanced preferred)
- Tutorial Flow

**Bonus points for:**

- DeFi related tutorial
- Including a video ([examples](https://www.youtube.com/playlist?list=PLkgTdjgP1aUCmcXrTxyRReS5OSLXlqGzB)) 
- Using DataHub ([Sign-up](https://datahub.figment.io/sign_up?service=celo))

### **Prize Breakdown**

- 1st place - $2,500 in cUSD + DataHub Plus for 3 months
- 2nd place - $1,500 in cUSD + DataHub Plus for 3 months
- 3rd place - $1,000 in cUSD + DataHub Plus for 3 months
- All approved submissions - $500 in cUSD

### **Helpful Resources**

- [Celo DataHub](https://datahub.figment.io/sign_up?service=celo)
- [Figment Learn](https://learn.figment.io/)
- [DataHub/Learn Discord](https://discord.gg/fszyM7K)
- [Submission Form](https://docs.google.com/forms/d/1lR6itLqtP8aTRuOPKSuQiF_m_m_J267nAifxuWKZlMs/edit?ts=604159f1&gxids=7628)
- [Official Celo Documentation](https://docs.celo.org/overview)

### **Submission Requirements**

First, take time to review [the guide we have created](https://docs.google.com/document/d/13LWLrWzZ34M0ldWGeDANcWxw9nEWk3AX3VwXRBIOs1M/edit) to help you structure your tutorial and make sure you follow the Learn standards. 

Then, please follow these steps to apply and start working on your tutorial. The tutorials publishing process is managed by the Figment Learn team. A review process is in place to ensure new tutorials are being looked at by developers, managers and documentation experts.

To get your new tutorial reviewed and published, please complete [this form](https://forms.gle/zCrfWmCppxY9TVFj6) and then email at: [yannick@figment.io](mailto:yannick@figment.io). The email should contain the following:

- Name
- Email
- Link to your Github profile
- Your tutorial based on which hackathon project
- Tutorial Topic & Title
- Programming language used

Once our team gets to it, they will review the content and open a discussion with you on discord or email. Once accepted you can follow the submission process: 

First, clone the [Datahub Learn documentation repository](https://github.com/figment-networks/datahub-learn): It's important that you have the most recent version of documentation before you start your contribution. If you have already cloned it, make sure to pull down the latest changes.

Create a new git branch in your local repository, you can name it however you want, though we prefer if you pick a name relevant to your contribution. (Example: `celo-defi-tutorial`).

Create a new file(s) under the tutorials section of the network documentation you're contributing to. Example: `network-documentation/celo/tutorials/my-new-tutorial.md`

Once you have completed the documentation make sure you have added a new entry in the `SUMMARY.md` file. 

Finally, when you're ready to submit your contribution for review, head over to the [Pull Requests](https://github.com/figment-networks/datahub-learn/pulls) section and create a new pull request from your branch. Our team will start the review process and suggest edits in the same pull request.

Once your pull request is approved and merged by our team, it will be automatically published on Learn!

### **Dates**

- Submissions Due - April 14th
- Winners announced - April 22nd

_Compensation may take a week after submission is accepted._

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 5000.0 cUSD (5000.0 USD @ $1.0/cUSD)  attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/figment-networks/datahub-learn/30/100024977).
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/figment-networks/datahub-learn/30/100024977).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://discord.com/invite/83BK5z55yB/'>Gitcoin's Discord</a>
* $2,577,574.04 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Work has been started__.


These users each claimed they can complete the work by 1 month, 1 week from now.
Please review their action plans below:


**1) [dslee111](https://gitcoin.co/dslee111) has started work.**

A Decentralized Web 3.0 Platform for Exchanging Knowledge. Publishing and sharing knowledge through PocketKnowledge™ NFT, without losing ownership control and giving up privacy.​

Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/figment-networks/datahub-learn/30/100024977).

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Work for 5000.0 cUSD (4965.06 USD @ $0.99/cUSD) has been submitted by__:

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/figment-networks/datahub-learn/30/100024977)
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/figment-networks/datahub-learn/30/100024977).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://discord.com/invite/83BK5z55yB/'>Gitcoin's Discord</a>
* $1,460,712.04 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Work for 5000.0 cUSD (4996.78 USD @ $1.0/cUSD) has been submitted by__:

 1. [@az111z2](https://gitcoin.co/az111z2)



@claytonfigmentnetworks please take a look at the submitted work:
* [PR](https://gitcoin.co/townsquare/?cb=ref:1f062) by @az111z2

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/figment-networks/datahub-learn/30/100024977)
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/figment-networks/datahub-learn/30/100024977).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://discord.com/invite/83BK5z55yB/'>Gitcoin's Discord</a>
* $1,701,405.70 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by pyr0gan** [comment]
I cannot see any work having been done, there is no associated GitHub account to this username, and the profile has a contributor rank of 1, while their submission is a copypaste job and there is at LEAST one other account with similar, suspicious credibility and a very similar name. See the attached screenshot which highlights the offending text.

It is my opinion that this is not a legitimate claim on the bounty and should be denied, possibly for TOS violations.
![gitcoinbot](https://user-images.githubusercontent.com/2707197/113361668-b4120280-9309-11eb-8ff6-472dbc9f97e5.png)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__Work for 5000.0 cUSD (5000.0 USD @ $1.0/cUSD) has been submitted by__:

 1. [@az111z2](https://gitcoin.co/az111z2)
 2. [@az111z](https://gitcoin.co/az111z)



@claytonfigmentnetworks please take a look at the submitted work:
* [PR](https://gitcoin.co/townsquare/?cb=ref:1f062) by @az111z
* [PR](https://gitcoin.co/townsquare/?cb=ref:1f062) by @az111z2

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/figment-networks/datahub-learn/30/100024977)
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/figment-networks/datahub-learn/30/100024977).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://discord.com/invite/83BK5z55yB/'>Gitcoin's Discord</a>
* $1,946,378.95 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__The funding of 5000.0 cUSD (5050.00 USD @ $1.01/cUSD)  attached to this issue has been approved & issued to @emmaodia.__  

 
* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/figment-networks/datahub-learn/30/100024977)
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://discord.com/invite/83BK5z55yB/'>Gitcoin's Discord</a>
* $2,001,651.39 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__The funding of 5000.0 cUSD (5050.00 USD @ $1.01/cUSD)  attached to this issue has been approved & issued to @emmaodia.__  

 
* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/figment-networks/datahub-learn/30/100024977)
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://discord.com/invite/83BK5z55yB/'>Gitcoin's Discord</a>
* $1,996,601.39 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
